meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx8mmevk/bootloader.uuu.in

```diff
@@ -11,6 +11,9 @@ FB: ucmd setenv mmcdev ${emmc_dev}
 FB: ucmd mmc dev ${emmc_dev}
 FB: flash bootloader ../imx-boot-@@MACHINE@@
 FB: flash bootloader2 ../u-boot-@@MACHINE@@.itb
+FB: flash bootloader_s ../imx-boot-@@MACHINE@@
+FB: flash bootloader2_s ../u-boot-@@MACHINE@@.itb
+FB: flash sit ../sit.bin
```

Comment:
Where are we generating sit.bin?

Reply:
yeah, I should have added a recipe for sit bin generation
Will do